<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentifyDlg</class>
 <widget class="QDialog" name="IdentifyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IdentifyDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wgt_title{
background:rgb(0, 144, 216);
}
#wgt_main{
background:gray;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wgt_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#btn_return{
	border-image: url(:/AIOViewer/Resources/return.png);
}
#lbl_title{
font-family:&quot;微软雅黑&quot;;
font-size:30px;
color:white;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>56</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_return">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>44</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>44</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_title">
        <property name="text">
         <string>人脸认证</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgt_main" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
        <property name="spacing">
         <number>60</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_collect">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>240</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_collect_2">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>240</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>240</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lbl_tips">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:18px;
color:#FFF;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="SuperButton" name="btn_collect">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SuperButton</class>
   <extends>QPushButton</extends>
   <header>SuperButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="AIOViewer.qrc"/>
 </resources>
 <connections/>
</ui>
